plugins/woocommerce/tests/e2e-pw/tests/shopper/checkout-block.spec.js

```diff
@@ -690,10 +685,12 @@ test.describe( 'Checkout Block page', () => {
 			.fill( customer.password );
 		await page.locator( 'text=Log in' ).click();
 		await expect(
-			page.getByRole( 'heading', { name: checkoutBlockPageTitle } )
+			page.getByRole( 'heading', { name: testPage.title } )
 		).toBeVisible();
 
 		// if edit address is present click it, otherwise fill shipping details
+		// eslint-disable-next-line playwright/no-networkidle
+		await page.waitForLoadState( 'networkidle' ); // need to wait for the page to load
 		if (
```

Comment:
I know you did not add this in your PR, but I'm not a fan of this conditional. We should know if the customer has a saved address or not and use the proper flow. If the address is being changed by other tests maybe we should use a unique customer per test so that we control the state.
I think the `networkidle` wait you added before is also because of this, and removing the conditional would also solve that?